src/app/interfaces/dataset.interface.ts

```diff
@@ -36,7 +38,6 @@ export interface Dataset {
   usedbychildren: ZfsProperty<number>;
   usedbydataset: ZfsProperty<number>;
   usedbyrefreservation: ZfsProperty<number>;
-  usedbysnapshots: ZfsProperty<number>;
```

Comment:
Is it been removed from the API response?

Reply:
@denysbutenko - I don't think so, but we will not use it on the FE side, do you think we should keep it?

Reply:
yes please